multisig/simulations/vip-004/vip-004-sepolia/simulations.ts

```diff
@@ -0,0 +1,44 @@
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { ethers } from "hardhat";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { forking, pretendExecutingVip } from "../../../../src/vip-framework/index";
+import { vip004 } from "../../../proposals/vip-004/vip-004-sepolia";
+import XVS_STORE_ABI from "./abi/xvsstore.json";
+import XVS_VAULT_ABI from "./abi/xvsvault.json";
+
+const { sepolia } = NETWORK_ADDRESSES;
+
+forking(4750277, () => {
+  let xvsVault: Contract;
+  let xvsStore: Contract;
+
+  before(async () => {
+    xvsVault = await ethers.getContractAt(XVS_VAULT_ABI, sepolia.XVS_VAULT_PROXY);
```

Comment:
If we include deployments in the hardhat config we can remove the need to supply the address and pull it from the deployments

Reply:
Something similar to the ABIs point: addresses can change in the npm package, and the simulations should be valid because they are usually "linked" to a specific block number. So, I would keep the addresses hardcoded.

But 100% open to talk about this deeper. We could also talk if we want to maintain a centralised file with every addresses (that would be similar (and worse) to use the npm packages), or we should define the address on each VIP file